Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallContentCommand.cs

```diff
@@ -187,30 +189,44 @@ private void InstallFile(SourceFile sourceFile, CancellationToken cancellationTo
                 DebugLogger.LogFormat("Destination file {0} already exists, removing it.", destinationFilePath);
                 FileOperations.Delete(destinationFilePath, cancellationToken);
             }
-
-            FileOperations.Move(sourceFile.FullPath, destinationFilePath, cancellationToken);
+#if UNITY_STANDALONE_WIN
+            if (destinationFilePath.Length > 259)
+            {
+                throw new InstallerException(string.Format("Cannot install file {0}, path is too length.", destinationFilePath)); 
```

Comment:
Rename `"Cannot install file {0}, path is too length."` to "Cannot install file {0}, the destination path length has exceeded Windows path length limit (260)."

Reply:
done